src/Raven.Server/Raven.Server.csproj

```diff
@@ -117,5 +117,6 @@
     <ServerGarbageCollection>true</ServerGarbageCollection>
     <ConcurrentGarbageCollection>true</ConcurrentGarbageCollection>
     <RetainVMGarbageCollection>true</RetainVMGarbageCollection>
+    <LangVersion>7.3</LangVersion>
```

Comment:
Can we revert this?

Reply:
we are using spans, so likely not.
In general, in 4.2, we are using the latest